src/__tests__/__snapshots__/fjs_babel-test.js.snap

```diff
@@ -137,6 +135,27 @@ foo[test()][bar()];
 "
 `;
 
+exports[`test ObjectExpression-ObjectPattern-Property.js 1`] = `
+"var foo = {};
+
+var foo = { x, y };
+
+var foo = { x: x, y: y };
+
+var foo = { x: x, y: y };
+
+var foo = {
+  [\'bar\']: \'foo\',
```

Comment:
I dont understand how it decides to escape things :/

But the good news is we have real computed properties now :D